scripts/attach_ebs_cache.sh

```diff
@@ -87,8 +87,17 @@ while [ "$(aws ec2 describe-volumes \
   elapsed_time=$((elapsed_time + WAIT_INTERVAL))
 done
 
-# Attach volume to the instance
+# First, make sure this is detached from any instances stuck in stopping state
+aws ec2 detach-volume \
+  --region $REGION \
+  --volume-id $VOLUME_ID || true
 
+# Force detach after for good measure, if this is attached to an instance actually doing a build we have other problems
+aws ec2 detach-volume \
+  --region $REGION \
+  --volume-id $VOLUME_ID --force || true
```

Comment:
ngl, force sounds scary:

> Forces detachment if the previous detachment attempt did not occur cleanly (for example, logging into an instance, unmounting the volume, and detaching normally). This option can lead to data loss or a corrupted file system. Use this option only as a last resort to detach a volume from a failed instance. The instance won't have an opportunity to flush file system caches or file system metadata. If you use this option, you must perform file system check and repair procedures.

Reply:
That's fair. I can leave it to just detach